package.json

```diff
@@ -136,5 +139,12 @@
     "**/terser/source-map": "0.7.4"
   },
   "version": "0.0.0",
-  "name": "sentry-javascript"
+  "name": "sentry-javascript",
+  "prettier": {
+    "arrowParens": "avoid",
+    "printWidth": 120,
+    "proseWrap": "always",
+    "singleQuote": true,
+    "trailingComma": "all"
+  }
```

Comment:
Low prio: Wdyt about putting this into a `.prettierrc`. Imo, that way it will be a bit more obvious that _we have_ a prettier config.

Reply:
I personally find it really hard to navigate when there is too much tooling specific configuration files in the root folder. If this is a non-blocker, I prefer to not do it. But I'm flexible.